tests/testsuite/kubevirtresource.go

```diff
@@ -103,6 +103,7 @@ func AdjustKubeVirtResource() {
 		virtconfig.WorkloadEncryptionSEV,
 		virtconfig.VMExportGate,
 		virtconfig.VSOCKGate,
+		virtconfig.PSASeccompAllowsUserfaultfd,
```

Comment:
If we use our own profile we no longer need to run migration postcopy e2e tests in a privileged namespace, I think this is the right PR to change them to use the default test namespace.

tests/migration_test.go:641
tests/migration_test.go:2147

Reply:
There are more things that can be removed after this PR. I would keep it for follow-up if you don't object.

Reply:
sounds good to me.